src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest

```diff
@@ -3,7 +3,7 @@
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
     <Identity Id="4db2d63d-3320-4fbd-bf80-07f8d1500bd3" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Microsoft"  />
-    <DisplayName>Microsoft Code Analysis 2017</DisplayName>
+    <DisplayName>Microsoft Code Analysis 2019</DisplayName>
     <Description xml:space="preserve">Live code analysis rules and code fixes addressing API design, performance, security, and best practices for C# and Visual Basic.</Description>
     <MoreInfo>https://go.microsoft.com/fwlink/?linkid=849061</MoreInfo>
```

Comment:
Tagging @gewarren - would you be able to help in creating a new fwlink for the "Microsoft Code Analysis 2019" VSIX? Current one points to VS2017 release. We also need to eventually update the documentation at https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019#to-install-fxcop-analyzers-as-a-vsix. I can file a new tracking docs issue for the same.

Reply:
I am actually going to replace the fwlink with https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019#to-install-fxcop-analyzers-as-a-vsix so we don't need to produce such an fwlink with every release. The documentation would need to be updated with each release to point to all available VSIX releases.

Reply:
Yes I will update that page for VS 2019. Thanks for letting me know.